elasticsearch_dsl/faceted_search_base.py

```diff
@@ -173,27 +173,26 @@ def __init__(
 
     def get_value_filter(self, filter_value: FilterValueType) -> Query:
         f, t = self._ranges[filter_value]
-        limits = {}
+        limits: Dict[str, Any] = {}
         if f is not None:
             limits["gte"] = f
         if t is not None:
             limits["lt"] = t
 
-        return Range(_expand__to_dot=False, **{self._params["field"]: limits})
```

Comment:
The `Range` query class now has a more reasonable format that is typing friendly, which uses `Range(field, value, **additional_kwargs)`. The old form works, but makes mypy unhappy.